### PR TITLE
 FEATURE: TMA-1052 Change parameters passing and create execution script for each brick

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/610/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1433/ : FAILURE in 1m 30s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1702/ : SUCCESS in 2m 58s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/620/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1444/ : FAILURE in 1m 29s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1710/ : FAILURE in 2m 07s

File: bin/run_brick.rb [hunk 1]
```diff
@@ -4,22 +4,22 @@
 require_relative '../lib/gooddata'
 
 DEFAULT_BRICK = 'hello_world_brick'
+BRICK_PARAM_PREFIX = 'BRICK_PARAM_'
 
-brick_type = ENV['BRICK_TYPE'] || DEFAULT_BRICK
+brick_type = ARGV.length > 0 ? ARGV[0] : DEFAULT_BRICK
```
Comment (yenkins) [inline bin/run_brick.rb]: warning: Style/ZeroLengthPredicate: Use `!empty?` instead of `length > 0`. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/621/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1445/ : FAILURE in 1m 25s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1711/ : SUCCESS in 1m 22s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/652/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1480/ : FAILURE in 2m 09s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1741/ : SUCCESS in 3m 21s

**Pull request merged**